ecal-readout-sim/src/main/java/org/hps/readout/ecal/updated/EcalDigitizationReadoutDriver.java

```diff
@@ -48,7 +48,7 @@ public EcalDigitizationReadoutDriver() {
         setTriggerPathTruthRelationsCollectionName("TriggerPathTruthRelations");
         setReadoutHitCollectionName("EcalReadoutHits");
         
-        setPhotoelectronsPerMeV(32.8);
+        setPhotoelectronsPerMeV(30.8);

```

Comment:
This constant should be pulled out into a variable and there should probably be a setter on the Driver for it.

Reply:
> We think that the default value 32.8 is wrong since typo mistake. Let's change the default value as 30.8. Otherwise, we need to reset all readout steering files.

Undocumented constants like this are a bad practice. I would suggest that this value be documented someplace within the code. Best way is probably putting the value into a variable with some documentation on it. The value could be a constant if it should never be changed by the user.

Reply:
I think that the value will not be changed unless Ecal group re-investigate Ecal noise and crystal light yields.